redhat-developer/app-services-cli
# fix(login): clear MAS tokens from config when using token login

Closes #594

### Verification Steps

1. Log in using an offline token: `rhoas login --token=$TOKEN`
2. Run `rhoas kafka topic list`.
3. You should get the following error message:

```shell
Error: not logged in to MAS-SSO. Run "rhoas login" to authenticate. Note: token-based login is not supported by the Kafka admin commands.
```

### Type of change


### Checklist
